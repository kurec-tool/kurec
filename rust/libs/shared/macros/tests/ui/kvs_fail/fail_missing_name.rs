//! bucket_name 属性がない場合にコンパイルエラーになることを確認

use shared_macros::define_kvs_bucket;

#[define_kvs_bucket] // bucket_name がない
struct MissingNameKv;

fn main() {}
