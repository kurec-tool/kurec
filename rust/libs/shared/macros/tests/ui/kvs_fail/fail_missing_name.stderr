error: Missing required attribute `bucket_name`
 --> tests/ui/kvs_fail/fail_missing_name.rs:5:1
  |
5 | #[define_kvs_bucket] // bucket_name がない
  | ^^^^^^^^^^^^^^^^^^^^
  |
  = note: this error originates in the attribute macro `define_kvs_bucket` (in Nightly builds, run with -Z macro-backtrace for more info)
